pkgs/development/python-modules/sqlcipher3/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  sqlcipher,
+  openssl,
+}:
+let
+  pname = "sqlcipher3";
+  version = "0.5.4";
+in
+buildPythonPackage {
+  inherit pname version;
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-4w/1jWTdQ+Gezt3RARahonrR2YiMxCRcdfK9qbA4Tnc=";
+  };
+
+  nativeBuildInputs = [
```

Comment:
```suggestion
  build-system = [
```

Reply:
based on the terms this looks good.
Unfortunately I couldn't figure out by reading on different sites like the [the manual](https://nixos.org/manual/nixpkgs/unstable/#var-stdenv-buildInputs) what really the difference is.
I thing I've found "build-system" only in the pythonPackaging-context while nativeBuildInputs looks like something more general.
If anyone has time I would like to read why one is more preferable then the other (newbie speaking).

Reply:
I'm sorry for not getting back to you sooner. I was overwhelmed by the notification.
You are correct that `build-system` is python builder-specific.

You can find more information in the python section, https://nixos.org/manual/nixpkgs/unstable/#handling-dependencies